.kokoro/presubmit/directpath-it.cfg

```diff
@@ -0,0 +1,38 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Configure the docker image for kokoro-trampoline.
+env_vars: {
+  key: "TRAMPOLINE_IMAGE"
+  value: "gcr.io/cloud-devrel-kokoro-resources/java8"
+}
+
+env_vars: {
+    key: "INTEGRATION_TEST_ARGS"
+    value: "-P bigtable-emulator-it,bigtable-prod-it -Dbigtable.project=bt-testing-dp -Dbigtable.instance=google-cloud-bigtable -Dbigtable.table=integration-tests"
```

Comment:
I think you want to replace `bigtable-emulator-it,bigtable-prod-it` with  `bigtable-directpath-it`